packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -120,6 +97,38 @@ contract Deploy is Deployer {
         }
     }
 
+    /// @notice Deploy all of the proxies
+    function deployProxies() public {
+        deployAddressManager();
+        deployProxyAdmin();
```

Comment:
Slight preference to rip these out of the proxy-specific function and keep them top level.
Maybe not so much the proxy admin, but the address manager at least.